src/pkg/cli/debug.go

```diff
@@ -27,7 +29,16 @@ var (
 	patterns            = []string{"*.js", "*.ts", "*.py", "*.go", "requirements.txt", "package.json", "go.mod"} // TODO: add patterns for other languages
 )
 
-func InteractiveDebug(ctx context.Context, c client.FabricClient, p client.Provider, etag types.ETag, project *compose.Project, failedServices []string) error {
+type DebugConfig struct {
+	Etag           types.ETag
+	Client         client.FabricClient
+	FailedServices []string
+	Project        *compose.Project
+	Provider       client.Provider
+	Since          time.Time
+}
+
+func InteractiveDebug(ctx context.Context, c client.FabricClient, p client.Provider, etag types.ETag, project *compose.Project, failedServices []string, since time.Time) error {
```

Comment:
Let's push the debugConfig up to the caller

```suggestion
func InteractiveDebug(ctx context.Context, debugConfig DebugConfig) error {
```